frontend/src/router.js

```diff
@@ -221,6 +236,12 @@ export function campFromRoute (route) {
   }
 }
 
+export function campCollaborationsFromInviteKey (inviteKey) {
+  return function () {
+    return this.api.get('/camp-collaborations?inviteKey=' + inviteKey)
```

Comment:
This hardcoded API route should be replaced with a templated URI on the API root, like we already have with `/camps/{id}`, `/scheduleEntries/{id}` etc.

Reply:
done